src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -17,10 +17,13 @@ import ScrollableForm from "components/ScrollableForm";
 import { useSupportedFeatures } from "context/useSupportedFeatures";
 import { LxdConfigPair } from "types/config";
 import { ensureEditMode } from "util/instanceEdit";
+import StoragePoolSelector from "pages/storage/StoragePoolSelector";
+import ResourceLink from "components/ResourceLink";
 
 export interface ProjectDetailsFormValues {
   name: string;
   description?: string;
+  default_instance_storage_pool?: string;
```

Comment:
```suggestion
  default_instance_storage_pool: string;
```
I think as we initialize this as string, we can make the type a bit stricter, and that will save some casts below.